### PR TITLE
Template README and license

### PR DESCRIPTION
**Pull request by mbbroberg**
* Including an open and permission reuse license (MIT)
* Structuring the README for our given audience and mission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
...
+- Connect to social media APIs (LinkedIn, Twitter) to allow for a quick template of people's stories to be automatically generated.
+- Support a mentoring function where experts can be safely connected to mentees for a 15-minute informational discussion on career success.
+
+## Architecture 
+
+Todo
+
+## Contributing 
+
+We're early days. Feel free to pick up where we started or add more statistics to why this matters.
```
Comment (travisormsby) [inline README.md]: We are in**

File: README.md [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+
+- Technology can catapult people to the middle class by paying [well above the average wage](https://www.business.org/business/startup/highest-tech-salaries) compared to other jobs in respective areas.
+- Technology is an attractive career option for people with different backgrounds because its about continued learning as opposed to a specific background. 
+- [Research](https://blog.salesflare.com/diversity-in-tech-industry) on diversity and inclusion in technology shows that representation is essential to long-term belonging.
+- Representation is also a business imperative: A study [from McKinsey & Company](https://www.mckinsey.com/business-functions/organization/our-insights/why-diversity-matters) found that companies in the top quartile for gender or racial and ethnic diversity are “more likely to have financial returns above their national industry medians.”
+- Our goal of cnonecting people through our work will help with the personal networks. In 2016, [LinkedIn estimated](https://news.linkedin.com/2017/6/eighty-percent-of-professionals-consider-networking-important-to-career-success) that 70% of people were hired at a company where they had a connection.
```
Comment (travisormsby) [inline README.md]: connecting

**Review by travisormsby — CHANGES_REQUESTED** [review]
-sifora

File: README.md (hunk 2 above)
Comment (mbbroberg) [reply README.md]: s/cnonecting/connecting/

**Pull request merged**